src/main/java/de/szut/lf8_project/lf8/project/dto/ProjectCreateDto.java

```diff
@@ -0,0 +1,74 @@
+package de.szut.lf8_project.lf8.project.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+import de.szut.employees.EmployeeAPI;
+import de.szut.lf8_project.exceptionHandling.InvalidDataException;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.web.client.HttpClientErrorException;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.text.MessageFormat;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+@Getter
+@Setter
+public class ProjectCreateDto {
+
+    @Min(0)
+    @NotNull
+    private Long customerId;
+
+    @Min(0)
+    @NotNull
+    private Long responsibleEmployeeId;
+
+    @Min(0)
+    @NotNull
+    private Long responsibleCustomerEmployeeId;
+
+    @Size(min = 1)
+    private List<Long> employees;
+
+    @Size(min = 1)
+    private List<String> skillSet;
+
+    @Size(min = 1)
+    @NotNull
+    private String description;
+
+    private LocalDate startDate;
+    private LocalDate endDate;
+
+    public ProjectCreateDto(Long customerId, Long responsibleEmployeeId, Long responsibleCustomerEmployeeId, List<Long> employees, List<String> skillSet, String description, LocalDate startDate, LocalDate endDate) {
+        this.customerId = customerId;
+        this.responsibleEmployeeId = responsibleEmployeeId;
+        this.responsibleCustomerEmployeeId = responsibleCustomerEmployeeId;
+        this.employees = employees;
+        this.skillSet = skillSet;
+        this.description = description;
+        this.startDate = startDate;
+        this.endDate = endDate;
+
+        if((startDate != null && endDate != null) && startDate.isAfter(endDate))
+            throw new InvalidDataException("The dates are not correct");
+
+        if(employees != null)
```

Comment:
I would say here we should also double check if the list is actually empty or not 
```suggestion
        if(employees != null && !employees.isEmpty())
```